src/sentry/integrations/slack/notifications.py

```diff
@@ -164,7 +165,7 @@ def _notify_recipient(
             "link_names": 1,
             "unfurl_links": False,
             "unfurl_media": False,
-            "text": notification.get_notification_title(),
+            "text": notification.get_subject(shared_context),
```

Comment:
@NisanthanNanthakumar this is going to change the text for every notification that uses Slack, not just the digest email

Reply:
@NisanthanNanthakumar Sure, could we just pass `shared_context` to `get_notification_title` then?

Reply:
@NisanthanNanthakumar I think you need to update getsentry as well

Reply:
I'm pretty sure builds will fail without it because of the mismatch on arguments, this isn't JS lol